packages/contracts/contracts/PassportFactory.sol

```diff
@@ -18,9 +18,9 @@ contract PassportFactory is Ownable {
     address passport
   );
 
-  function create(string memory name_, string memory symbol_, uint256 supply, uint256 price, string memory metadataHash) public {
+  function create(string memory name_, string memory symbol_, uint256 supply, uint256 price, string memory metadataHash, bool claimable) public {
     require(supply > 0, "Required to mint at least 1 passport");
-    Passport passport = new Passport(msg.sender, name_, symbol_, supply, price, metadataHash);
+    Passport passport = new Passport(msg.sender, name_, symbol_, supply, price, metadataHash, claimable);
```

Comment:
I wonder what's the max number of parameters a constructor/method could handle in Solidity lol